packages/clay-css/src/scss/mixins/_alerts.scss

```diff
@@ -46,6 +46,8 @@
 
 	$hr: setter(map-get($map, hr), ());
 
+	$alert-indicator: setter(map-get($map, alert-indicator), ());
```

Comment:
@pat270 have we declared in `$map` the `alert-indicator`? for example in `$alert`, I saw that we have just separated it as a variable, or here we are just adding this option but we are not using it in our implementation for example.

Reply:
We are just adding an option and not using it. It gives devs a chance to overwrite our styles scoped to a specific variant.

Reply:
I thought about it, but decided not to because it would increase the specificity of the original [`.alert-indicator`](https://github.com/liferay/clay/blob/10ca4f01024811869141cea5dfc88a0de1033446/packages/clay-css/src/scss/components/_alerts.scss#L20-L28) to `.alert .alert-indicator`.

Reply:
Yeah, you're right, I had forgotten that we have the `.alert-indicator` for use in alert-feedback too.